Sources/Views/InputTextView.swift

```diff
@@ -198,6 +200,12 @@ open class InputTextView: UITextView {
         placeholderLabelConstraintSet?.right?.constant = -placeholderLabelInsets.right
     }
     
+    // MARK: - Notification
+    
+    private func postTextViewDidChangeNotification() {
+        NotificationCenter.default.post(name: .UITextViewTextDidChange, object: self)
```

Comment:
Do you think we need to post a notification here? What about keeping the behavior consistent with UIKit and just disabling the button in the `didSet`? I'm not saying this is the right approach. I want your opinion. cc @nathantannar4 please check this when you get a chance

Reply:
Emm, I think it needs. because besides adjust enabled of `sendButton`, we need also handle `placeholderLabel`,`messageInputBar`'s size, and so on.